.github/workflows/nightly_release.yml

```
@@ -0,0 +1,67 @@
+name: SalesforceCommerceCloud/pwa-kit/nightly_release
+on:
+  workflow_dispatch:
+  schedule:
+    # Run every day at 12am (PST) - cron uses UTC times
+    - cron: "0 8 * * *"
+
+jobs:
+  create_nightly_release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Get Timestamp
+        run: |-
+          echo "release_timestamp=$(date +'%m%d%Y%H%M%S')" >> "$GITHUB_ENV"
+
+      - name: Get monorepo version
+        run: |-
+          version=`cat package.json | jq -r ".version" | cut -d "-" -f 1`
```

```suggestion
          version=`jq -r ".version" package.json | cut -d "-" -f 1`
```
Another place we don't need 🐈‍⬛! 🤭

---

updated command to use jq instead of 🐈